src/pydash/__version__.py

```diff
@@ -2,4 +2,4 @@
 """Project version information.
 """
 
-__version__ = '4.8.0'
+__version__ = '4.8.1'
```

Comment:
Similar to the CHANGELOG comment, this is only bumped once a release is getting cut. So no need to change this now.